q2_phylogeny/tests/test_align_to_tree_mafft_iqtree.py

```diff
@@ -0,0 +1,42 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2019, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import unittest
+
+from qiime2.plugin.testing import TestPluginBase
+from qiime2 import Artifact
+
+
+class AlignToTreeMafftFasttreePipelineTest(TestPluginBase):
+    package = 'q2_phylogeny.tests'
+
+    def setUp(self):
+        super().setUp()
+        self.align_to_tree_mafft_iqtree = self.plugin.pipelines[
+                                    'align_to_tree_mafft_iqtree']
+
+        input_sequences_fp = self.get_data_path('dna-sequences-1.fasta')
+        self.input_sequences = Artifact.import_data('FeatureData[Sequence]',
+                                                    input_sequences_fp)
+
+    def test_execution(self):
+        # Does it run?
+        self.align_to_tree_mafft_iqtree(self.input_sequences)
```

Comment:
Can you add in a `self.assertTrue(True)` right after this? Just as a simply "checkpoint" to ensure we are actually testing a thing.

Reply:
Done.